docs/modules/idrac_network_attributes.rst

```diff
@@ -0,0 +1,348 @@
+    The maximum wait time of *job_wait* in seconds. The job is tracked only for this duration.
+
+    This option is applicable when *job_wait* is ``true``.
+
+
+  idrac_ip (True, str, None)
+    iDRAC IP Address.
+
+
+  idrac_user (True, str, None)
+    iDRAC username.
+
+
+  idrac_password (True, str, None)
+    iDRAC user password.
+
+
+  idrac_port (optional, int, 443)
+    iDRAC port.
+
+
+  validate_certs (optional, bool, True)
+    If ``false``, the SSL certificates will not be validated.
+
+    Configure ``false`` only on personally controlled sites where self-signed certificates are used.
+
+    Prior to collection version ``5.0.0``, the *validate_certs* is ``false`` by default.
+
+
+  ca_path (optional, path, None)
+    The Privacy Enhanced Mail (PEM) file that contains a CA certificate to be used for the validation.
+
+
+  timeout (optional, int, 30)
+    The socket level timeout in seconds.
+
+
+
+
+
+Notes
+-----
+
+.. note::
+   - Run this module from a system that has direct access to Dell iDRAC.
+   - This module supports both IPv4 and IPv6 address.
+   - This module supports ``check_mode``.
+
+
+
+
+Examples
+--------
+
+.. code-block:: yaml+jinja
+
+    
+    ---
+    - name: Configure OEM network attributes
+      dellemc.openmanage.idrac_network_attributes:
+        idrac_ip: "192.168.0.1"
+        idrac_user: "user_name"
+        idrac_password: "user_password"
+        network_id: "NIC.Integrated.1"
+        network_port_id: "NIC.Integrated.1-1-1"
+        apply_time: "Immediate"
+        oem_network_attributes:
+          IscsiInitiatorIpAddr: "192.168.1.0"
+
+    - name: Configure OEM network attributes to apply on reset
+      dellemc.openmanage.idrac_network_attributes:
+        idrac_ip: "192.168.0.1"
+        idrac_user: "user_name"
+        idrac_password: "user_password"
+        network_id: NIC.Integrated.1
+        network_port_id: "NIC.Integrated.1-1-1"
+        oem_network_attributes:
+          SNMP.1.AgentCommunity: public
+        apply_time: OnReset
+        maintenance_window:
+          start_time: "2022-09-30T05:15:40-05:00"
+          duration: 600
+
+    - name: Configure OEM network attributes to apply at maintainance window
+      dellemc.openmanage.idrac_network_attributes:
+        idrac_ip: "192.168.0.1"
+        idrac_user: "user_name"
+        idrac_password: "user_password"
+        network_id: NIC.Integrated.1
+        network_port_id: "NIC.Integrated.1-1-1"
+        oem_network_attributes:
+          SNMP.1.AgentCommunity: public
+        apply_time: AtMaintenanceWindowStart
+        maintenance_window:
+          start_time: "2022-09-30T05:15:40-05:00"
+          duration: 600
+
+    - name: Clearing the pending attributes
+      dellemc.openmanage.idrac_network_attributes:
+        idrac_ip: "192.168.0.1"
+        idrac_user: "user_name"
+        idrac_password: "user_password"
+        network_id: NIC.Integrated.1
+        network_port_id: "NIC.Integrated.1-1-1"
```

Comment:
does this example need an update as well?

Reply:
Done